Extensions/NotificationService/NotificationService.swift

```diff
@@ -20,46 +17,37 @@ class NotificationService: UNNotificationServiceExtension {
     // AppDelegate.application(_:didReceiveRemoteNotification:completionHandler:)
     // Once the notification is tapped, then the same userInfo is passed to the same method in the AppDelegate.
     override func didReceive(_ request: UNNotificationRequest, withContentHandler contentHandler: @escaping (UNNotificationContent) -> Void) {
-        DispatchQueue.global().async {
```

Comment:
Wait, what? I'm not even sure why we're using `Dispatch.main.async` in the first place.